index.html

```diff
@@ -1,29 +0,0 @@
-<!DOCTYPE html>
```

Comment:
you removed a file that needs to be existed

Reply:
I recreated the file